src/main/java/frc/team3388/robot/RobotMap.java

```diff
@@ -7,11 +7,7 @@
 
     // EXAMPLE SYSTEM
     public static final int EXAMPLE_SYSTEM_MOTOR = 5;
+    public static final int SHOOTER_SYSTEM_MOTOR=4;
     // HID
     public static final HidChannel XBOX = RoboRio.newHidChannel(0);
-    public static final int HOPPER_MOTOR = 6;
-    public static final int INTAKE_SYSTEM_MOTOR = 1;
-    public static final int PISTON_REVERSE_CHANNEL = 2;
-    public static final int PISTON_FORWARD_CHANNEL = 3;
-
```

Comment:
Did you remove these during the merge? You might've hit `X` instead of the arrow to accept the changed made by others.

This a problem.. Take those lines and manually paste them in your code.